.github/workflows/windows.yml

```
@@ -348,7 +349,7 @@ jobs:
       -
         name: Test CLI
         run: |
-          & gotestsum --format=standard-verbose --packages="./integration-cli/..." -- `
+          & gotestsum --format=standard-verbose --jsonfile="./bundles/coverage-report-int-cli-tests.json" --junitfile="./bundles/coverage-report-int-cli-tests.xml" --packages="./integration-cli/..." -- `
```

🙈 Just noticed now, after I downloaded the results

```suggestion
          & gotestsum --format=standard-verbose --jsonfile="./bundles/go-test-report-int-cli-tests.json" --junitfile="./bundles/junit-report-int-cli-tests.xml" --packages="./integration-cli/..." -- `
```

Is wrapping allowed here? Wondering ig wrapping it would make it more readable;

```suggestion
          & gotestsum `
                --format=standard-verbose `
                --jsonfile="./bundles/go-test-report-int-cli-tests.json" `
                --junitfile="./bundles/junit-report-int-cli-tests.xml" `
                --packages="./integration-cli/..." `
                -- `
```